.github/workflows/release-body-libs.yml

```diff
@@ -0,0 +1,101 @@
+    steps:
+      - name: Get latest release
+        uses: rez0n/actions-github-release@v2.0
+        id: latest_release
+        env:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          repository: ${{ github.repository }}
+          type: "stable"
+
+      - name: Get settings for this release
+        id: get_settings
+        shell: python
+        run: |
+          import os
+          import re
+          import sys
+
+          semver_no_meta = '''^(?P<major>0|[1-9]\d*)\.(?P<minor>0|[1-9]\d*)\.(?P<patch>0|[1-9]\d*)(?:-(?P<prerelease>(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?$'''
+          tag_name = '${{ github.event.release.tag_name }}'
+
+          is_valid_version = re.match(semver_no_meta, tag_name) is not None
+          if not is_valid_version:
+            print(f'Release version {tag_name} is not a valid full or pre-release. See RELEASE.md for more information.')
+            sys.exit(1)
+
+          is_prerelease = '-' in tag_name
+
+          # Safeguard: you need to both set "latest" in GH and not have suffixes to overwrite latest
+          is_latest = '${{ steps.latest_release.outputs.release }}' == tag_name and not is_prerelease
+
+          with open(os.environ['GITHUB_OUTPUT'], 'a') as ofp:
+            print(f'is_latest={is_latest}'.lower(), file=ofp)
+      
+  release-body-libs:
+    needs: [release-settings]
+    if: ${{ needs.release-settings.outputs.is_latest == 'true' }} # only for latest releases
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone repo
+        uses: actions/checkout@v4
+        
+      - name: Create release body file
+        run: |
+          touch release-body.md
+          
+      - name: Extract MIN_API version
+        run: |
+          MIN_API_VER=$(grep SCAP_MINIMUM_DRIVER_API_VERSION userspace/libscap/scap.h)
+          prefix="#define SCAP_MINIMUM_DRIVER_API_VERSION PPM_API_VERSION"
+          MIN_API_VER=${MIN_API_VER/#$prefix}
+          MIN_API_VER=$(echo $MIN_API_VER | tr -d "(" | tr -d ")")
+          MIN_API_VER=$(echo $MIN_API_VER | sed -r 's/, /./g')
+          echo '!'"[MIN_DRIVER_API](https://img.shields.io/badge/MIN_DRIVER_API-${MIN_API_VER}-yellow)" >> release-body.md
+          
+      - name: Extract MIN_SCHEMA version
+        run: |
+          MIN_SCHEMA_VER=$(grep SCAP_MINIMUM_DRIVER_SCHEMA_VERSION userspace/libscap/scap.h)
+          prefix="#define SCAP_MINIMUM_DRIVER_SCHEMA_VERSION PPM_API_VERSION"
+          MIN_SCHEMA_VER=${MIN_SCHEMA_VER/#$prefix}
+          MIN_SCHEMA_VER=$(echo $MIN_SCHEMA_VER | tr -d "(" | tr -d ")")
+          MIN_SCHEMA_VER=$(echo $MIN_SCHEMA_VER | sed -r 's/, /./g')
+          echo '!'"[MIN_DRIVER_SCHEMA](https://img.shields.io/badge/MIN_DRIVER_SCHEMA-${MIN_SCHEMA_VER}-yellow)" >> release-body.md
+          ECHO "" >> release-body.md
+          
+      - name: Generate release notes
```

Comment:
Note the steps from this one to the create release one can be refactored to a reusable_workflow since they are shared between driver and libs.

Reply:
I am not going to do that since it would involve uploading `release-body` artifact to download it inside the new reusable workflow; too much of a work to avoid repeating 3 steps.

It would be great if falcosecurity started providing an `actions` repo with many reusable actions like `build-falco`, `build-libs`, `release-body` etc etc.
I might work on this in the future.